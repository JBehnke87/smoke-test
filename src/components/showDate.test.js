import React from 'react';
import ShowDate from './ShowDate';
import renderer from 'react-test-renderer';

test('test show Date', () => {
  const tree = renderer.create( <ShowDate /> ).toJSON();
  expect(tree).toMatchSnapshot();
});