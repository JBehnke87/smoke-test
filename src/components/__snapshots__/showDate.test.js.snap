// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test show Date 1`] = `
<div>
  <span>
    The correct date is: 
    Wed Mar 31 2021 15:08:52 GMT+0200 (Mitteleurop√§ische Sommerzeit)
  </span>
  <span>
    The correct date is: 2017-04-05
  </span>
</div>
`;
